src/context/init.js

```diff
@@ -52,10 +54,12 @@ function initServices(context) {
   context.addInstance('errorHandler', errorHandler);
   context.addInstance('ipWhitelist', ipWhitelist);
   context.addInstance('forestServerRequester', forestServerRequester);
+  context.addInstance('superagentRequest', superagentRequest);
```

Comment:
[Suggestion] Looking at the server code, maybe this should be handled separately since it is a dependency. Maybe adding an `initExternals` block would make sense ? Also, you are missing the type hint